packages/suite-data/files/translations/en.json

```diff
@@ -963,6 +963,16 @@
   "TR_EXCHANGE_USE_NON_SUITE_ACCOUNT": "Use an account ({symbol}) that isn't in Suite",
   "TR_EXCHANGE_VERIFY_ADDRESS_STEP": "Receive address",
   "TR_EXCHANGE_VIEW_DETAILS": "View details",
+  "TR_COINMARKET_EXCHANGE_FIXED_OFFERS_HEADING": "Fixed-rate CEX",
```

Comment:
You mean `messages.ts`, right? `en.json` will be overwritten by crowdin anyway, is that true?

Reply:
yes messages.ts

Reply:
Has there been any change about that recently? Because before we just had to copy it into the en.json too. If it's not needed anymore, that's great :) (cc @LukasRada)